docs/content/admin/quickstart.mdx

```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
 
 ---
 
-## Instant Deployment to Netlify
+## Install the Admin
 
-Instead of manually following this guide to install then later deploy the Medusa Admin, you can deploy the Medusa Admin to Netlify with this button:
+### Step 1: Install the Package
 
-<a href="https://app.netlify.com/start/deploy?repository=https://github.com/medusajs/admin" className="img-url">
-  <img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify" className="no-zoom-img" />
-</a>
+In the directory of your Medusa backend, run the following command to install admin dashboard:
 
----
+```bash npm2yarn
+npm install @medusajs/admin
```

Comment:
note: if the user intends to deploy the admin separately, they can add the plugin as a `devDependency` instead, as they should only be using the build tool aspect and not inject any code that will be used at runtime.